_posts/2022-06-13-rails-discord-server-is-now-open-to-the-public.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Rails Discord server is now open to the public"
+categories: news
+author: rafaelfranca
+published: true
+date: 2022-06-13
+---
+
+In the past few weeks, the Rails team has been working on a few changes to lower the barrier for new contributors to the framework.
+
+We started by [disabling the automatic closing of stale PRs](https://github.com/rails/rails/commit/acf48169943011834c4c885b956e29050548fa98). This means that contributors can submit their PRs without
+the fear of a bot closing their worthy contributions before a member of the Rails Team has a chance to review them.
+
+To make it easier for contributors to recognize members of the Rails Team, we made public the [list
```

Comment:
I think this sentence is trying to say that listing the issues team members made it easier to recognize the "whole" team, which can be made clearer:
```suggestion
To make it easier for contributors to recognize all the members of the Rails team, we made public the [list
```